inner-scripts/kubestellar

```diff
@@ -84,7 +90,8 @@ function ensure_imws() {
                 fi
             fi
             run_kb_konnector_for_imw
-            bind_mangodbs_for_imw
+            bind_resource_for_imw "synctargets"
+            bind_resource_for_imw "locations"
             if ! kubectl get apibinding "edge.kubestellar.io" &> /dev/null; then
                 kubectl kcp bind apiexport root:espw:edge.kubestellar.io
```

Comment:
Do we leave for now both the kcp bind and kube-bind?

Reply:
Yes, that's the current status.